hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/StreamReadMonitoringFunction.java

```diff
@@ -226,9 +226,9 @@ public void monitorDirAndForwardSplits(SourceContext<MergeOnReadInputSplit> cont
     this.issuedOffset = result.getOffset();
     LOG.info("\n"
             + "------------------------------------------------------------\n"
-            + "---------- consumed to instant: {}\n"
+            + "---------- consumed {} to instant: {}\n"
             + "------------------------------------------------------------",
-        this.issuedInstant);
+        conf.getString(FlinkOptions.TABLE_NAME), this.issuedInstant);
```

Comment:
I would like it to be:

---------- table: xxx
---------- consumed to instant: xxx

Reply:
Done.